tests/test_placement.py

```diff
@@ -10,17 +10,18 @@
 
 import pytest
 
-from pytket.phir.machine import Machine
+from pytket.circuit import OpType
+from pytket.phir.machine import Machine, MachineTimings
 from pytket.phir.placement import (
     GateOpportunitiesError,
     InvalidParallelOpsError,
     place,
     placement_check,
 )
 
-m = Machine(4, {1}, 10, 2, 2)
-m2 = Machine(6, {1, 3}, 10, 2, 2)
-m3 = Machine(8, {0, 6}, 10, 2, 2)
+m = Machine(4, {OpType.TK1, OpType.TK2}, {1}, MachineTimings(10, 2, 2))
+m2 = Machine(6, {OpType.TK1, OpType.TK2}, {1, 3}, MachineTimings(10, 2, 2))
+m3 = Machine(8, {OpType.TK1, OpType.TK2}, {0, 6}, MachineTimings(10, 2, 2))
```

Comment:
I wonder if we can create a constant for this gate set with a descriptive name, perhaps even to be used as the default in the `Machine` class?

Reply:
This was an arbitrary choice -- other universal gatesets would have done as well -- but good idea to make it a constant to avoid repetition. I'm not sure it's worth making it a default for `Machine` -- though I agree it makes sense to make the quantinuum gateset the default for `QtmMachine`.

Reply:
> This was an arbitrary choice -- other universal gatesets would have done as well

Ah, I misunderstood it as being critical. In that case, I agree it's not required to be the default.